layout/paddedlayout.go

```diff
@@ -0,0 +1,43 @@
+// Package layout defines the various layouts available to Fyne apps
```

Comment:
I don't think the package comments should be duplicated - only appearing in one file in the package.

Reply:
fixed